src/LMS1xx_node.cpp

```diff
@@ -45,14 +45,9 @@ int main(int argc, char **argv)
 	outputRange = laser.getScanOutputRange();
       }
 
-      //check if laser is fully initialized, else reconnect
-      //assuming fully initialized => scaningFrequency=5000
-      if (cfg.scaningFrequency != 5000) {
```

Comment:
Maybe we should have an additional check instead, not forcing anything, but a sanity check to be completely sure the configuration is good.